README.md

```diff
@@ -62,16 +26,74 @@ These are the currently supported instrumentations with the minimum version requ
 | [mongodb/mongo-go-driver](contrib/mongodb/mongo-go-driver) | 1.0+ |
 | [net/http](contrib/net/http) | Standard Library |
 
-### API
-The API is documented on [godoc](https://godoc.org/github.com/signalfx/signalfx-go-tracing/tracing).
+Other libraries are available to instrument, but are in beta and aren't
+officially supported. You can view all the libraries in the
+[contrib](contrib) directory.
 
-### Testing
-Tests can be run locally using the Go toolset. The grpc.v12 integration will fail (and this is normal), because it covers for deprecated methods. In the CI environment
-we vendor this version of the library inside the integration. Under normal circumstances this is not something that we want to do, because users using this integration
-might be running versions different from the vendored one, creating hard to debug conflicts.
+## Configuration values
 
-To run integration tests locally, you should set the `INTEGRATION` environment variable. The dependencies of the integration tests are best run via Docker. To get an
-idea about the versions and the set-up take a look at our [CI config](https://github.com/signalfx/signalfx-go-tracing/blob/master/.circleci/config.yml).
+Set configuration values from environment variables or directly in your code:
 
-The best way to run the entire test suite is using the [CircleCI CLI](https://circleci.com/docs/2.0/local-jobs/). Simply run `circleci build`
-in the repository root. Note that you might have to increase the resources dedicated to Docker to around 4GB.
+| Code | Environment Variable | Default Value | Notes |
+| ---  | ---                  | ---           | ---   |
+| [WithServiceName](https://godoc.org/github.com/signalfx/signalfx-go-tracing/tracing/#WithServiceName) | `SIGNALFX_SERVICE_NAME` | `SignalFx-Tracing` | The name of the service. |
+| [WithEndpointURL](https://godoc.org/github.com/signalfx/signalfx-go-tracing/tracing/#WithEndpointURL) | `SIGNALFX_ENDPOINT_URL` | `http://localhost:9080/v1/trace` | The URL to send traces to.  |
+| [WithAccessToken](https://godoc.org/github.com/signalfx/signalfx-go-tracing/tracing/#WithAccessToken) | `SIGNALFX_ACCESS_TOKEN` | none | The access token for your SignalFx organization. |
+| [WithGlobalTag](https://godoc.org/github.com/signalfx/signalfx-go-tracing/tracing/#WithGlobalTag) | `SIGNALFX_SPAN_TAGS` | none | Comma-separated list of tags included in every reported span. For example, "key1:val1,key2:val2". Use only string values for tags.|
+
+## Instrument a Go application
```

Comment:
I think we should split this into two sections. 

1. `Setup tracer & instrument application`
This should cover adding dependencies, initializing the tracer and one example of a manually generated span.

2. Auto-instrumentation example and list of supported packages.

Reply:
what does an auto-instrumentation example look like?

Reply:
This section actually is about auto-instrumentation. I'm thinking perhaps we should separately show three things.

1. Installing and setting up the tracer
2. Creating manual spans (not in this doc right now)
3. Using any of the provided auto-instrumentation packages (what this section covers, specifically 2.0)

That said, it would make more sense if all the languages followed this pattern and not sure if we want to make that change now. //cc @rmfitzpatrick @johnbley

Reply:
that sounds out of scope for this PR.